src/main/webapp/app/exercises/shared/import/exercise-import-wrapper/exercise-import-wrapper.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { ExerciseType } from 'app/entities/exercise.model';
+import { ProgrammingLanguage } from 'app/entities/programming-exercise.model';
+
+@Component({
+    selector: 'jhi-exercise-import-wrapper',
+    templateUrl: './exercise-import-wrapper.component.html',
+})
+export class ExerciseImportWrapperComponent implements OnInit {
+    readonly ExerciseType = ExerciseType;
+
+    @Input()
+    exerciseType: ExerciseType;
+    titleKey: string;
+    @Input()
+    programmingLanguage?: ProgrammingLanguage;
+
+    constructor(private activeModal: NgbActiveModal) {}
+
+    ngOnInit(): void {
+        if (this.programmingLanguage) {
+            this.titleKey = 'artemisApp.programmingExercise.configureGrading.categories.importLabel';
+        } else {
+            this.titleKey =
+                this.exerciseType === ExerciseType.FILE_UPLOAD ? `artemisApp.fileUploadExercise.home.importLabel` : `artemisApp.${this.exerciseType}Exercise.home.importLabel`;
+        }
```

Comment:
```suggestion
        if (this.programmingLanguage) {
            this.titleKey = 'artemisApp.programmingExercise.configureGrading.categories.importLabel';
        } else if (this.exerciseType === ExerciseType.FILE_UPLOAD) {
            this.titleKey = 'artemisApp.fileUploadExercise.home.importLabel';
        } else {
            this.titleKey = `artemisApp.${this.exerciseType}Exercise.home.importLabel`;
        }
```
This would be more readable as an if-elseif-else I think